cloudbaseinit/metadata/services/vmwareguestinfoservice.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 ruilopes.com
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import base64
+import os
+
+from cloudbaseinit import exception
+from cloudbaseinit.metadata.services import base
+from cloudbaseinit.osutils import factory as osutils_factory
+
+RPCTOOL_PATH = 'C:/Program Files/VMware/VMware Tools/rpctool.exe'
```

Comment:
this path needs to be made configurable, you can set its default value to this path

Reply:
Done!